### DISCUSSION TITLE
Option to Reset Camera Trims

### DISCUSSION THREAD

**Discussion by sca075** [sca075/mqtt_vacuum_camera#207] [Show and tell]
## What's New
As per when the vacuums are scanning again the floor the map trims will not update automatically as before (saving time at boot was the reason for this change). In the case the vacuum is scanning again the floor as soon it is done will be possible to reset the trim stored data from the "Advanced Options" of the camera.

![Screenshot 2024-07-17 at 12 04 07](https://github.com/user-attachments/assets/7c7cb3a3-0230-4677-8764-b03f372c2c09)

**Once the option is selected, as soon done, it is necessary to reload the camera, from the dots on the right side of the camera**

![Screenshot 2024-07-17 at 22 38 46](https://github.com/user-attachments/assets/2d7133c2-9742-44b4-a7db-92f2af705374)

## What's Changed
* Resolving issue #205  (@sca075 in #206)

**Full Changelog**: https://github.com/sca075/mqtt_vacuum_camera/compare/2024.07.1...2024.07.2

<hr /><em>This discussion was created from the release <a href='https://github.com/sca075/mqtt_vacuum_camera/releases/tag/2024.07.2'>Option to Reset Camera Trims</a>.</em>

**Comment by sca075** [comment]
So.. Indeed maintaining a project (small like this) is a quite challenge, trying to fix things as fast I can as unusual leads to some "mistake" that generally I try to avoid with test after test. There anyhow some "quality check" procedure to improve in my development  environment and I will try to fix those in the next days.. If you face any issue with the Camera please be so kind to report this and I will do my best to address it. 

The **2024.07.2** had a big issue.. it wan't startup because I forgot to update to key files, therefore if you already downloaded it I'm really sorry for this. I did correct either the 2024.07.2 and release **2024.07.3** to solve the issue with the missing _LOGGER declaration in const.py. 

In summary the purpose of the last updates is to keep the Camera updated and working in HA without any issue, as well I'm trying to improve the code to make it a little more.. easy to read and perform better. 

Any help, is really appreciated. Also suggestions and / or proposals are welcome, but more than other.. as the scope of this project is to make thinks easier.. please speak out when something isn't working as expected :) 
I'm only human after all so I can make mistakes :) 

Use it like, it and if you want start it ;)
